bench/bench.go

```diff
@@ -70,7 +70,7 @@ func (s *scaleOut) Run() error {
 	}
 	s.t.addTime = time.Now()
 	for {
-		time.Sleep(time.Second)
+		time.Sleep(time.Second * 10)
```

Comment:
Maybe make it a variable?

Reply:
or we will introduce a config file?

Reply:
Will it depend on data size or machine nums, I think check per second is ok?

Reply:
In order to avoid the expansion does not take effect, I will add an additional number of checks later